rakefile

```diff
@@ -47,7 +47,7 @@ end
 # TODO: replace original version and remove '_awsdirect'
```

Comment:
I think you don't need this todo anymore?

Reply:
I was thinking that when we're no longer using the old endpoint these new versions would replace the current versions; no reason to keep the '_awsdirect' designation at that point:

performance_awsdirect >> performance
performance_provision_with_abs_awsdirect >> performance_provision_with_abs
performance_deprovision_with_abs_awsdirect >> performance_deprovision_with_abs

If we do want to keep the current versions and have the option of using either endpoint I'll remove the todos.